### PR TITLE
data:Madhavi210

### PR DESCRIPTION
**Pull request by Madhavi210**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3623"></a>

### CODE REVIEW

File: data/Madhavi210.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Madhavi_Joshi__",
+      "icon": "FaTwitter"
+    }
+     {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/madhavi-joshi-0b3228239/",
+      "icon": "FaLinkedIn"
```
Comment (senali-d) [inline data/Madhavi210.json:16]: Linkedin icon name should be `FaLinkedin`

```suggestion
      "icon": "FaLinkedin"
```

**Review by senali-d — CHANGES_REQUESTED** [review]

File: data/Madhavi210.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Learning Full Stack Web Development",
+  "avatar": "https://github.com/Madhavi210.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Madhavi_Joshi__",
+      "icon": "FaTwitter"
+    }
```
Comment (ChinmayMhatre) [inline data/Madhavi210.json:12]:
```suggestion
    },
```
missing comma

Comment (senali-d) [inline data/Madhavi210.json:12]: Missing `,`

File: data/Madhavi210.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+     {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/madhavi-joshi-0b3228239/",
+      "icon": "FaLinkedIn"
+    }
```
Comment (ChinmayMhatre) [inline data/Madhavi210.json:17]:
```suggestion
    },
```
Missing comma

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Please add the commas as suggested

File: data/Madhavi210.json (hunk 3 above)
Comment (senali-d) [inline data/Madhavi210.json:17]: Missing `,`
```suggestion
    },
```

**Review by senali-d — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**